cli/tsc/mod.rs

```diff
@@ -444,6 +443,9 @@ pub fn as_ts_script_kind(media_type: MediaType) -> i32 {
   }
 }
 
+pub const MISSING_DEPENDENCY_SPECIFIER: &str =
+  "internal:///missing_dependency.d.ts";
```

Comment:
This is the source of a lot of confusion in Deno when dependencies can't resolve, but it ends up as any.